AdvancedNewFile.sublime-settings

```diff
@@ -165,6 +165,10 @@
     // the move command.
     "warn_overwrite_on_move": false,
 
+    // If a warning should be displayed when trying to overwrite an existing file using
+    // the copy command.
+    "warn_overwrite_on_copy": false
```

Comment:
Missing comma at the end.
```suggestion
    "warn_overwrite_on_copy": false,
```

Reply:
@FichteFoll do you think a better default will be to set the flag as true (less dangerous by default)

Reply:
Generally, yes, and the same would apply to the move option as well. Although I would go a step further and remove the options altogether since I really don't see a good reason to disable that behavior.

I intend to do a more thorough cleanup of the code base at some point where we I batch a couple breaking changes together and also bump the minimum required ST version to 4, but I can't say when I'll find the time for that. For now, let's stick with this default for symmetry with the move setting.